recipes/dmgbuild/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "dmgbuild" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7ced2603d684e29c22b4cd507d1e15a1907e91b86259924b8cfe480d80553b43
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  entry_points:
+    - dmgbuild = dmgbuild.__main__:main
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+    - setuptools
+  run:
+    - python >=3.9
+    - importlib_resources  # [py<39]
```

Comment:
If we are building py>=39 we don't need this. If you do want to build for py38 you can change the constraints above but remove the selector here, `importlib_resources` is just a stub in py>=3.9, so it is fine to install it there.

```suggestion
```